db/migrate/.20231019171723_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :role, :string
```

Comment:
- [ ] Good job so far by adding this column but you should make sure you update the schema file, in the schema file users table doesn’t have the role column 🙃 🙃 
- [ ] As there is no role column we can't create posts, add comments, and like, kindly fix that 😁 

<img width="771" alt="Screenshot 2023-10-20 at 18 25 31" src="https://github.com/ctokoli/Blog/assets/74501523/fd6f9c3f-b1af-411a-92aa-8226539f9196">